### ISSUE TITLE
TE3 unable to automatically activate license when upgrading versions

### ISSUE THREAD

**Issue by alwortma** [TabularEditor/TabularEditor3#608]
Our team has started to encounter an issue that we _believe_ started around v3.3.0.

When upgrading versions most functionality is disabled (we have Enterprise licenses), and _NOT ACTIVATED_ appears at the top of the screen.

We receive a prompt to perform manual activation:
![image](https://user-images.githubusercontent.com/97995966/179615184-2a43865c-0131-4f3a-a456-dd4396200a1e.png)

Clicking the link does not open a window in my browser (Edge), but I can get the activation key by pasting the link into my browser, then copying the activation code.

Activation does work after doing this, but it's just annoying because we encourage the team to always update to the latest version available as you guys keep rolling out new functionality

**Comment by alwortma** [comment]
Yes we are behind a proxy. I was just fiddling with my proxy settings when I was trying to get the enhancement from #604 working. Let me try downgrading then upgrading again with different proxy settings.

**Comment by otykier** [comment]
Instead of downgrading then upgrading you can reset your license key as described [here](https://docs.tabulareditor.com/te3/getting-started.html#changing-a-license-key-through-the-registry), to force TE3 to attempt to activate upon next launch.

**Comment by alwortma** [comment]
Somewhat counter-intuitive...but setting Proxy Type to _None_ fixes the issue.
